src/exec/run-batch-submitter.ts

```diff
@@ -170,12 +179,18 @@ export const run = async () => {
               nonce: i,
             })
             log.info(`Submitting transaction with nonce: ${i}; hash: ${response.hash}`)
+
+            const resubmissionConfig = {
+              numConfirmations: parseInt(requiredEnvVars.NUM_CONFIRMATIONS, 10),
+              resubmissionTimeout: 60 * 1_000,  // Attempt resubmission every 60 seconds
```

Comment:
@karlfloersch @tynes refactoring things into optional env vars right now, is there a reason we don't use the config here?

Reply:
1) definitely think that `resubmissionTimeout` should be configurable, and 2) I definitely think that the batch submitter configuration is lackluster *at best*. We need to do a bigger refactor & for specifics @tynes is the config master

Reply:
@karlfloersch it is configurable, but this particular line uses a different number than the config so i was wondering why

Reply:
extra context, when we initialize the stateBatchSubmitter, we pass in the config resubmission timeout:
https://github.com/ethereum-optimism/batch-submitter/blob/7997688603152eb231a334c2c1b72770c6f8b5fc/src/exec/run-batch-submitter.ts#L147

but here we use a specific value that seems like it may be different, or should it actually be what's in the config?

Reply:
We do need to refactor the config, this library here has everything that we would ever want + no additional random deps: https://github.com/bcoin-org/bcfg

> but here we use a specific value that seems like it may be different, or should it actually be what's in the config?

Here we want the transaction to go through as soon as possible because we are replacing the expensive tx (gaswise) in the mempool with a simple send to ourselves. This unsticks the transaction from the mempool and increments the nonce so that the account can be used to send more transactions, since the nonce must increment one by one for a transaction to be valid